src/hooks/player_management/assign_physical_index.cpp

```diff
@@ -85,6 +85,21 @@ namespace big
 				{
 					if (plyr->get_net_data()->m_gamer_handle.m_rockstar_id != 0)
 					{
+						auto ip   = plyr->get_ip_address();
+						auto text = std::format("{}.{}.{}.{}", ip.m_field1, ip.m_field2, ip.m_field3, ip.m_field4);
```

Comment:
Good that you predefine the variables, makes it readable. This one could be used on line 95 as well.

Reply:
Thank you, I will implement this later